xDrip/Extensions/UView+Tests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UView+Tests.swift
```

Comment:
This file is used by unit tests target only, so move it to `xDripTests` instead of `xDrip`.